test_bugbear.py

```
@@ -0,0 +1,132 @@
+    
+    # B010: Do not call setattr with a constant attribute
+    setattr(obj, 'attr', 'value')  # B010
+    
+    # B011: Do not call assert False since python -O removes these calls
+    assert False  # B011
+    
+    # B012: Use of break/continue/return in finally blocks
+    try:
+        pass
+    finally:
+        break  # B012
```

Ruff (0.12.2)

`break` outside loop

(F701)